src/main/java/com/google/api/codegen/transformer/InitCodeTransformer.java

```diff
@@ -936,4 +937,25 @@ private static String getVariableName(MethodContext context, InitCodeNode item)
     }
     return context.getNamer().localVarName(item.getIdentifier());
   }
+
+  private static String getCliFlagDefaultValue(InitCodeNode item) {
+    checkArgument(
+        !item.getType().isMessage(), "Only enums and primitive types are supported for now.");
+
+    String value = item.getInitValueConfig().getInitialValue().getValue();
+
+    if (item.getType().isStringType()) {
```

Comment:
Are these two conditions captured in a test?

Reply:
No. I'll add a few tests.